### PR TITLE
Integrated ContextMenu form @datapunt/asc-ui and added the correct in…

### PR DESCRIPTION
**Pull request by royderks**
…tegration with the current codebase

### CODE REVIEW

File: src/panorama/components/PanoramaToggle/PanoramaToggle.jsx [hunk 1]
```diff
@@ -1,112 +1,96 @@
...
+  openPanoramaTags,
+  openPanoramaExternal
+}) => {
+  const [showMenu, showMenuToggle] = React.useState(null);
 
-  onOpenPanoramaExternal() {
-    this.props.openPanoramaExternal();
-    this.toggleMenu();
-  }
+  React.useEffect(() => {
```
Comment (aditudorache) [inline src/panorama/components/PanoramaToggle/PanoramaToggle.jsx]: Nice improvement!

File: src/panorama/components/StatusBar/StatusBar.jsx [hunk 2]
```diff
@@ -33,10 +30,8 @@ StatusBar.defaultProps = {
 };
 
 StatusBar.propTypes = {
-  heading: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
   date: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Date)]).isRequired,
-  currentLabel: PropTypes.string.isRequired,
-  location: PropTypes.array.isRequired //eslint-disable-line
+  location: PropTypes.instanceOf(Array).isRequired //eslint-disable-line
```
Comment (aditudorache) [inline src/panorama/components/StatusBar/StatusBar.jsx]: remove eslint rule

File: src/shared/styles/components/_button.scss [hunk 3]
```diff
@@ -40,7 +40,7 @@ $icon-height: 24px;
   @markup
     <button class="btn btn--link">Lorem ipsum</button>
   */
-  background-color: transparent;
+  // background-color: transparent;
```
Comment (aditudorache) [inline src/shared/styles/components/_button.scss]: remove

File: src/app/components/ContextMenu/ContextMenu.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
+@import "../../../shared/styles/config";
+
+.context-menu {
```
[the remark this answers is not in the document]
Comment (royderks) [reply src/app/components/ContextMenu/ContextMenu.scss]: This is specific styling like the location on the page and focus styling (which for now should match the legend on the map for instance)

File: src/map/containers/map/MapContainer.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
+@import "../../../shared/styles/config";
+
+.bottom-left {
```
[the remark this answers is not in the document]
Comment (royderks) [reply src/map/containers/map/MapContainer.scss]: This was refactored and should be removed

File: src/map/containers/map/MapContainer.scss [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  bottom: $container-whitespace;
+  display: flex;
+  left: $container-whitespace;
+  position: absolute;
+  z-index: 9;
```
[the remark this answers is not in the document]
Comment (royderks) [reply src/map/containers/map/MapContainer.scss]: This was refactored and should be removed

**Pull request merged**